banzai_nres/rv.py

```diff
@@ -75,8 +76,14 @@ def cross_correlate_over_traces(image, orders_to_use, velocities, template):
         # Only pass in the given wavelength range +- 1 Angstrom to boost performance
         order_indices = np.logical_and(template['wavelength'] >= np.min(order['wavelength']) - 1.0,
```

Comment:
The variable name "order_indices" doesn't match what this variable actually is. "order_indices" seems to indicate that this is the order number. A better name might be "relevant_region" or "template_overlap" or similar.